### DISCUSSION TITLE
skoala/registry/managed/governance/governance-policy/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1575] [Announcements]
# skoala/registry/managed/governance/governance-policy/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/registry/managed/governance/governance-policy/

**Comment by liyuerich** [comment]
还有个集群流控，能介绍下吗？
